open-metadata-implementation/repository-services/repository-services-implementation/src/main/java/org/odpi/openmetadata/repositoryservices/enterprise/repositoryconnector/executors/AddEntityProxyExecutor.java

```diff
@@ -0,0 +1,53 @@
+/* SPDX-License-Identifier: Apache 2.0 */
```

Comment:
I don't think this is needed since the callers of the enterprise connector will never call addEntityProxy()